addons/portal/views/portal_templates.xml

```diff
@@ -73,7 +73,7 @@
                 <t t-if="_avatar">
                     <t t-if="user_id.image_256" t-set="avatar_source" t-value="image_data_uri(user_id.image_256)"/>
                     <t t-else="" t-set="avatar_source" t-value="'/web/static/src/img/placeholder.png'"/>
-                    <img t-att-src="avatar_source" t-attf-class="rounded-circle #{_avatar_class}" width="24" height="24" alt="" loading="eager"/>
+                    <img t-att-src="avatar_source" t-attf-class="rounded-circle #{_avatar_class}" alt="" loading="eager"/>
```

Comment:
Why bother passing `_avatar_class`? Couldn't add class name here all the time?

I would suggest to name it `portal-user-dropdown-avatar` instead to better match the actual template name.

Reply:
> Why bother passing `_avatar_class`? Couldn't add class name here all the time?
> 
> I would suggest to name it `portal-user-dropdown-avatar` instead to better match the actual template name.

@seb-odoo ok bro